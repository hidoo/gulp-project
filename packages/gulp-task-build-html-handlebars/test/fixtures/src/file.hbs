---

---
depth: {{this.path.depth}}
relative: {{this.path.relative}}
basename: {{this.path.basename}}
extname: {{this.path.extname}}
