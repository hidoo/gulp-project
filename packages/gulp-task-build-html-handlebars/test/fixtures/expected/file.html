depth: ./
relative: file.html
basename: file.html
extname: .html
